packages/perseus/src/components/math-input.tsx

```diff
@@ -32,13 +32,14 @@ import type {Keys, MathFieldInterface} from "@khanacademy/math-input";
 
 type ButtonsVisibleType = "always" | "never" | "focused";
 
-type KeypadButtonSets = {
+export type KeypadButtonSets = {
```

Comment:
This export was mainly for supporting tests - I figured it would be better to use the existing `KeypadButtonSets` type than creating a new one in the test suite and having to remember to keep it in sync with the real one